app/src/main/java/org/wikipedia/theme/ThemeChooserDialog.kt

```diff
@@ -86,7 +86,12 @@ class ThemeChooserDialog : ExtendedBottomSheetDialogFragment() {
         updateComponents()
         disableBackgroundDim()
         requireDialog().window?.let {
-            DeviceUtil.setNavigationBarColor(it, getThemedColor(requireContext(), R.attr.paper_color))
+            DeviceUtil.setNavigationBarColor(it, ResourceUtil.getThemedColor(requireContext(), R.attr.paper_color))
+        }
+
+        // TODO: test only
```

Comment:
Is this still a TODO?

Reply:
Removed in my PR @dbrant https://github.com/wikimedia/apps-android-wikipedia/pull/3060/files#diff-c1eae10dbe3c866e252b9d1e4a1386098e0313ee4b3e23c11f23d02360b2ab59L92